Lib/logging/__init__.py

```diff
@@ -360,6 +366,9 @@ def __init__(self, name, level, pathname, lineno,
             self.process = os.getpid()
         else:
             self.process = None
+        if logAsyncioTasks:
+            aio = sys.modules.get('asyncio')
+            self.taskname = aio.current_task().get_name()
```

Comment:
- If `asyncio` hasn't been imported, `aio` is `None` and this will fail with `AttributeError`.
- If we aren't running any asyncio task right now, `current_task()` will return `None` and we again get `AttributeError`.

In other words, this fails with or without the asyncio import:

```python
#import asyncio
import logging
logging.error("blah blah")
```

I think we should also make sure that `self.taskname` is always set to something, and just set it to `None` if `logAsyncTasks` is set to `False` or no asyncio task is running. See how it sets `self.processName = None` above.